tenets/codelingo/code-review-comments/FULL_TEXT.md

```diff
@@ -606,6 +606,6 @@ In any case, the onus is on you to fail with a helpful message to whoever's debu
 
 ## Variable Names
 
-Variable names in Go should be short rather than long.  This is especially true for local variables with limited scope.  Prefer `c` to `lineCount`.  Prefer `i` to `sliceIndex`.
+Variable names in Go should be short rather than long.  This is especially true for local variables with limited scope ([Tenet](https://github.com/codelingo/codelingo/tree/master/tenets/codelingo/code-review-comments/short-name-limited-scope). Prefer `c` to `lineCount`.  Prefer `i` to `sliceIndex`.
```

Comment:
```suggestion
Variable names in Go should be short rather than long.  This is especially true for local variables with limited scope ([Tenet])(https://github.com/codelingo/codelingo/tree/master/tenets/codelingo/code-review-comments/short-name-limited-scope). Prefer `c` to `lineCount`.  Prefer `i` to `sliceIndex`.
```